src/Domains/Inventory/Importer/Jobs/ProductImporterJob.php

```diff
@@ -146,6 +154,21 @@ public function handle()
                 'finished_at' => now(),
             ]);
         }
+        $subscriptionData = [
```

Comment:
@FredPeal move this part to a sperate function withing the job and you should be good